<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-8L8BQUV" failures="1" tests="3" name="Personal loan" time="0.053" errors="0" timestamp="2024-07-24T15:03:28 WEST">
  <testcase classname="testNGTutorial.day_2" name="SecondTest" time="0.004"/>
  <testcase classname="testNGTutorial.day_2" name="ThirdsTest" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGTutorial.day_2.ThirdsTest(day_2.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- ThirdsTest -->
  <testcase classname="testNGTutorial.day_1" name="Demo" time="0.001"/>
  <testcase classname="testNGTutorial.day_1" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal loan -->
